src/main/java/core/mvc/asis/DispatcherServlet.java

```diff
@@ -40,6 +53,17 @@ protected void service(HttpServletRequest req, HttpServletResponse resp) throws
         }
     }
 
+    private void serviceAnnotationMapping(HttpServletRequest req, HttpServletResponse resp) throws ServletException {
+        HandlerExecution handlerExecution = annotationHandlerMapping.getHandler(req);
+        try {
+            ModelAndView modelAndView = handlerExecution.handle(req, resp);
+            move(modelAndView, req, resp);
```

Comment:
```suggestion
            move(modelAndView.getViewName(), req, resp);
```

실제 구현 코드는 기존의 move 메서드와 동일해보이는데요. 별도로 작성하신 이유가 따로 있으실까요?

```java
        RequestDispatcher rd = req.getRequestDispatcher(viewName);
        rd.forward(req, resp);
```

Reply:
현재는 기존 move에 있던 내용과 View구현제인 JspView의 render안의 내용이 같지만 나중에 View구현체가 추가 되었을때 rendor의 구현이 달라질 수 있다고 생각해서 별도로 작성했습니다. 
이부분은 RequestMapping과 AnnotationHandlerMapping class 에 대한 Controller추상화가 진행되면 하나의 move메소드로 합쳐질 듯 합니다.